src/ethereum_test_types/transaction_types.py

```diff
@@ -652,36 +655,58 @@ class NetworkWrappedTransaction(CamelModel, RLPSerializable):
     """
     Network wrapped transaction as defined in
     [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844#networking).
+
+    < Osaka:
+        rlp([tx_payload_body,                   blobs, commitments, proofs])
+
+    >= Osaka:
+        rlp([tx_payload_body, wrapper_version,  blobs, commitments, cell_proofs])
     """
 
     tx: Transaction
-    wrapper_version: Literal[1] | None = None
-    blobs: Sequence[Blob]
+    blob_objects: Sequence[Blob]
+    wrapper_version: Union[Bytes | None] = None  # only exists in >= osaka
```

Comment:
Thanks for removing the changes to `src/ethereum_test_base_types/conversions.py`, but this still needs a fix, run this in the python cli to see what I mean:
```python
from ethereum_test_types import NetworkWrappedTransaction, Transaction
NetworkWrappedTransaction(tx=Transaction(), blob_objects=[], wrapper_version=1)
```

My suggestion is to just roll it back to an `int` or `None`
```suggestion
    wrapper_version: Literal[1] | None = None  # only exists in >= osaka
```

Or the other approaches could be `Bytes` or `None` and use a pydantic validator, or change `tx_wrapper_version` in `tests/osaka/eip7594_peerdas/test_get_blobs.py` to return `b"\1"`.